rust/agama-server/src/web/service.rs

```diff
@@ -84,7 +84,7 @@ impl MainServiceBuilder {
             .route("/ping", get(super::http::ping))
             .route("/auth", post(login).get(session).delete(logout));
 
-        let serve = ServeDir::new(self.public_dir);
+        let serve = ServeDir::new(self.public_dir).precompressed_gzip();
```

Comment:
> Both the precompressed version and the uncompressed version are expected to be present in the directory.

https://docs.rs/tower-http/latest/tower_http/services/struct.ServeDir.html#method.precompressed_gzip

That means we will later probably need to update the packaging of the web frontend files to contain both files.

Reply:
Yes, exactly.